pydantic_ai_slim/pydantic_ai/_agent_graph.py

```diff
@@ -621,7 +621,7 @@ async def process_function_tools(  # noqa: C901
                 result_data = await toolset.call_tool(call, run_context)
             except exceptions.UnexpectedModelBehavior as e:
                 ctx.state.increment_retries(ctx.deps.max_result_retries, e)
-                raise e
+                raise  # pragma: no cover
```

Comment:
I think `raise e` is more clear in this context as we have `e` as a variable anyway

Reply:
It not being covered is unexpected?

Reply:
Restored the `e`, I removed as it's recommend to no included by ruff strict settings, as mentioned in the other PR.

I believe this was being flagged as covered due to the python 3.9 line numbering bug, but now that's disabled to fix other issues, it's actually not covered.